keras/backend/jax/numpy.py

```diff
@@ -6,8 +6,6 @@
 
 
 def add(x1, x2):
-    x1 = convert_to_tensor(x1)
-    x2 = convert_to_tensor(x2)
```

Comment:
Removing the convert_to_tensor statements will cause bugs. They are there for a reason (to enable JAX to process Keras variables as arguments)

Reply:
Thanks for pointing that out. I removed them unintentionally.